modules/users/client/views/invite.client.view.html

```diff
@@ -27,6 +27,33 @@
           Codes are valid for 24 hours.
         </p>
 
+        <!-- Go to www.addthis.com/dashboard to customize your tools -->
+        <p>
+          <div ng-if="::invite.invitation.code">
+          <script type="text/javascript">
+            var code = document.getElementById("invite").value.toUpperCase();
+            var addthis_config = addthis_config||{};
+                addthis_config.ui_email_from = 'help@trustroots.org';
+                addthis_config.ui_email_note = 'Your invite code: ' + code + ' would be shared which can be used for signup with Trustroots at https://www.trustroots.org/signup?code=' + code;
+
+            var addthis_share = {
+              url: 'https://www.trustroots.org/signup?code=' + code,
+              title: 'Invite by ' + user.displayName + ' to join Trustroots',
+              description: 'You have been invited to join Trustroots, an invite only community for travelers. Your invite code to join Trustroots is: ' + code,
+              email_template: 'new_template',
+              email_vars: { 'code': code, 'invitee': user.displayName },
+              passthrough: {
+                twitter: {
+                  text: 'You have been invited to join Trustroots. Your invite code: ' + code + ' and url:'
+                }
```

Comment:
I just found out where to modify these and went on and made it look like this:

![image](https://cloud.githubusercontent.com/assets/87168/26450321/9148ef02-415f-11e7-9170-9ef07d9949b5.png)